addons/mail/wizard/mail_compose_message.py

```diff
@@ -701,18 +704,33 @@ def _action_send_mail_mass_mail(self, res_ids, auto_commit=False):
         return mails_sudo
 
     def action_save_as_template(self):
```

Comment:
Maybe time to rename like
```suggestion
    def action_save_as_mail_template(self):
```